apps/mull-ui/src/app/components/event-card/event-card.tsx

```diff
@@ -22,7 +22,7 @@ export const EventCard = ({ event, style = {} }: EventCardProps) => {
   // TODO: Implement distance calculation
   const distance = 15;
 
-  const { day, month, time } = formatDate(event.startDate);
+  const { day, month, time } = formatDate(new Date(event.startDate));
```

Comment:
I ran into this issue too where `startDate` and `endDate` aren't really dates but are rather strings. Maybe the event service should be tweaked to fix this, so that querying for dates doesn't give back strings. What do you think @r-saba ?

Reply:
The API gives us a serialized object and it's the responsibility of the front-end to deserialize it. This is why I added the `Date | string` type to the interface below https://github.com/RGPosadas/Mull/pull/129#discussion_r548065726. 

An option I was considering is another interface such as `ISerializedEvent` to store the returned data and then once we seralize the event we can use `IEvent`. What do you think, any suggestions?